ingest/.mypy.ini

```diff
@@ -0,0 +1,8 @@
+[mypy]
+python_version = 3.12
+
+[mypy-orjsonl]
+ignore_missing_imports = True
+
+[mypy-pandas.*]
+ignore_missing_imports = True
```

Comment:
I think we could just import the stubs for it? Then we don't need to ignore imports!

Reply:
how do we do this? (I also did this in prepro- might be wrong)